modules/clients/src/test/java/org/apache/ignite/internal/processors/rest/JettyRestProcessorAbstractSelfTest.java

```diff
@@ -2955,8 +2980,12 @@ public VisorGatewayArgument forNodes(Collection<ClusterNode> nodes) {
          * @return This helper for chaining method calls.
          */
         public VisorGatewayArgument arguments(Object... vals) {
-            for (Object val : vals)
-                put("p" + idx++, String.valueOf(val));
+            if (vals != null) {
```

Comment:
What if `vals` will be empty array? May be we should use `!F.isEmpty(vals)` ?

Reply:
You'll get a link to memory section, e.g. _[I@2ff4acd0_, a default object behavior.

It was always like this in the past, is that wrong?

Reply:
Added checks on methods of VisorGatewayArtument class to prevent errors in time of test creation.
Some methods of VisorGatewayArtument are renamed.